### DISCUSSION TITLE
Documentation on how to custumoze internal default keybindings

### DISCUSSION THREAD

**Discussion by azzamsa** [abougouffa/minemacs#82] [Q&A] [answered]
Hi.

Is there any recommended approach to customize `+minemacs--internal-map!` https://github.com/abougouffa/minemacs/blob/16db93aa9da21c946d83a530fff24219a67ad2ef/core/me-keybindings.el#L68?

Such as changing/overriding the default keybinding.

Thanks! :heart:

**Comment by azzamsa** [comment]
I did this in my `config.el`. It works but do you have any better approach?

```elisp
;;
;; Bring back favorite Doom's behavior
;;

;; SPC RET: bookmark-jump
(with-eval-after-load 'consult
  (+map!
    "RET" #'consult-bookmark
    ))

```

**Reply by abougouffa** [reply, marked as answer]
At the end of `general`'s `use-package` block, I provide `me-general-ready`. If you do something like:

```elisp
;; SPC RET: bookmark-jump
(with-eval-after-load 'me-general-ready
  (+map!
    "RET" #'consult-bookmark
    ))
```

It should work as expected.
